platforms/jvm/testing-jvm/src/integTest/groovy/org/gradle/testing/testng/TestNGTestFrameworkIntegrationTest.groovy

```diff
@@ -88,9 +88,6 @@ class TestNGTestFrameworkIntegrationTest extends AbstractTestFrameworkIntegratio
         """
 
         when:
-        executer.expectDocumentedDeprecationWarning("No test executed. This behavior has been deprecated. " +
-            "This will fail with an error in Gradle 9.0. There are test sources present but no test was executed. Please check your test configuration. " +
-            "Consult the upgrading guide for further information: https://docs.gradle.org/current/userguide/upgrading_version_8.html#test_task_fail_on_no_test_executed")
```

Comment:
This is a similar problem with TestNG where tests are discovered, but since they're disabled, we were triggering the deprecation warning, even though this scenario is not what the deprecation warning is intended to catch.  We'll no longer get the warning in this case either.